crates/sui-core/src/authority_active/checkpoint_driver/mod.rs

```diff
@@ -285,35 +284,14 @@ pub async fn checkpoint_process<A>(
 
         let proposal = state_checkpoints.lock().set_proposal(committee.epoch);
 
-        // (4) Check if we need to advance to the next checkpoint, in case >2/3
```

Comment:
This is indeed correct to delete -- we do not need to wait to see 2/3 proposals at each validator to start making fragments. The process by which we make fragments starts with this node making a proposal (we should check for that?) and then alliteratively asking other random (weighted) validators for their proposal to make and sequence a fragment at a time.